features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -1080,160 +529,760 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
+                && (isSubOrganization() ?
+                    !brandingDisabledFeatures.includes(
+                        BrandingPreferencesConstants.APP_WISE_BRANDING_FEATURE_TAG): true)
+                && !isMyAccount
+            ) {
+                if (applicationConfig.editApplication.
+                    isTabEnabledForApp(
+                        inboundProtocolConfig?.oidc?.clientId,
+                        ApplicationTabTypes.GENERAL,
+                        tenantDomain
+                    )) {
+                    panes.push({
+                        componentId: "general",
+                        "data-tabid": ApplicationTabIDs.GENERAL,
+                        menuItem:
+                                 <Menu.Item data-tourid="general">
+                                     { t("applications:edit.sections.general.tabName") }
+                                 </Menu.Item>,
+                        render: () =>
+                            applicationConfig.editApplication.
+                                getOveriddenTab(
+                                    inboundProtocolConfig?.oidc?.clientId,
+                                    ApplicationTabTypes.GENERAL,
+                                    GeneralApplicationSettingsTabPane(),
+                                    application?.name,
+                                    application?.id,
+                                    tenantDomain
+                                )
+                    });
+                }
+            }
+            if (isFeatureEnabled(featureConfig?.applications,
+                ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_ACCESS_CONFIG"))
+                && !isFragmentApp
+                && !isMyAccount
+            ) {
+
+                applicationConfig.editApplication.isTabEnabledForApp(
+                    inboundProtocolConfig?.oidc?.clientId,
+                    ApplicationTabTypes.PROTOCOL,
+                    tenantDomain
+                ) &&
+                panes.push({
+                    componentId: "protocol",
+                    "data-tabid": ApplicationTabIDs.PROTOCOL,
+                    menuItem: t("applications:edit.sections.access.tabName"),
+                    render: ApplicationSettingsTabPane
+                });
+            }
+            if (isFeatureEnabled(featureConfig?.applications,
+                ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_ATTRIBUTE_MAPPING"))
+                && !isFragmentApp
+                && !isM2MApplication
+                && (UIConfig?.legacyMode?.applicationSystemAppsSettings ||
+                    application?.name !== ApplicationManagementConstants.MY_ACCOUNT_APP_NAME)) {
+
+                applicationConfig.editApplication.isTabEnabledForApp(
+                    inboundProtocolConfig?.oidc?.clientId, ApplicationTabTypes.USER_ATTRIBUTES, tenantDomain) &&
+                panes.push({
```

Comment:
```suggestion

                panes.push({
```

Reply:
This is an existing code block. https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/components/edit-application.tsx#L312-L314